spec/decorators/course_decorator_spec.rb

```diff
@@ -83,4 +83,80 @@
       end
     end
   end
+
+  context "status tag" do
+    let(:status_tag) { course.decorate.status_tag }
+
+    context "A non running course" do
+      let(:course) { build(:course, ucas_status: "not_running") }
+
+      it "Returns nil" do
```

Comment:
How many courses does this affect? not_running ought to be the same as withdrawn.